server/integration/endpoint/src/test/java/org/infinispan/server/endpoint/EndpointSubsystemTestCase.java

```diff
@@ -68,7 +68,7 @@ public EndpointSubsystemTestCase(String xmlFile, int operations, String xsdPath)
       Object[][] data = new Object[][] {
             { "endpoint-7.2.xml", 16, "schema/jboss-infinispan-endpoint_7_2.xsd" },
             { "endpoint-8.0.xml", 16, "schema/jboss-infinispan-endpoint_8_0.xsd" },
-            { "endpoint-9.0.xml", 37, "schema/jboss-infinispan-endpoint_9_0.xsd" },
+            { "endpoint-9.0.xml", 38, null },
```

Comment:
Yes, because with @ryanemerson 's changes to add parametrized defaults, the schema at this point is invalid (it contains ${} expressions where typed defaults should be)